test/hack/e2e_scripts/install_karpenter.sh

```diff
@@ -32,3 +32,4 @@ helm upgrade --install karpenter "${CHART}" \
   --set "serviceMonitor.endpointConfig.relabelings[3].targetLabel=commitsAfterTag" \
   --set "serviceMonitor.endpointConfig.relabelings[3].replacement=\"$(git describe --tags | cut -d '-' -f 2)\"" \
   --wait
+  --timeout 8m0s
```

Comment:
default is `5m0s` - this could be a red herring and things are failing to install so potentially the timeout is just failing for longer

Reply:
Yeah, I think our builds were just broken right now. Should be fixed now so if you run again, I think that should fix it.